### DISCUSSION TITLE
demande d'extension de deadline

### DISCUSSION THREAD

**Discussion by jfcouchot** [danymat/INSAnonym-utils#69] [General]
reBonjour.
On corrige quelques bugs pour être compatible avec les métriques.
Serait-il possible d'avoir une extension de deadline de quelques heures ?
JF pour FemtOrange.

**Comment by beng-git** [comment]
Oui pas de soucis :)

**Reply by jfcouchot** [reply]
Nous avons soumis un fichier cet après midi et j'ai dû partir.
Entre temps, la compétition a été fermée.
S'il ne contient pas d'erreur, serait-il possible d'avoir le score de ce fichier et le publier ?
Merci.
JF
